guides/common/modules/con_how-content-view-environments-are-handled-during-registration-with-multiple-activation-keys.adoc

```diff
@@ -0,0 +1,49 @@
+[id="how-content-view-environments-are-handled-during-registration-with-multiple-activation-keys"]
+= How content view environments are handled during registration with multiple activation keys
```

Comment:
This heading is too long.

How about _Multiple activation keys and content environments_?

Reply:
Yeah it should be shorter. I like your suggestion. I was also thinking _Content view environments with multiple activation keys_. But I think yours is more clear.